docker/Dockerfile

```diff
@@ -10,10 +10,22 @@
 ################################################################################
 
 FROM centos:7 AS prep_open_search_files
+ARG JDK_VER=11
```

Comment:
Do we need this? As this will automatically set to 11 from workflow logic.

Reply:
Yes, you're right, from the workflow point of view it doesn't do much. I left it as a default value in the case someone deploys this locally, because we use the same Dockerfile [here](https://github.com/opensearch-project/performance-analyzer-rca/blob/main/INSTALL.md?plain=1#L94) so that docker build can be called without build args.